hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetadataManager.java

```diff
@@ -116,6 +116,25 @@
    */
   String getOzoneDirKey(String volume, String bucket, String key);
 
+  /**
+   * Given a volume, bucket and a key, return the corresponding DB key.
+   *
```

Comment:
The Comment needs to be updated, here we don't pass volume, bucket and key. The same problem in getOzoneLeafNodeKey method.

Reply:
Will fix it in new commit.